packages/schema/lib/schemas/AuthInputFieldSchema.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+const FieldSchema = require('./FieldSchema');
+
+module.exports = makeSchema(
+  {
+    ...FieldSchema.schema,
+    id: '/AuthInputFieldSchema',
+    description:
+      'Field schema specialized for auth input fields (e.g., OAuth).',
+    properties: {
+      ...FieldSchema.schema.properties,
+      isSafe: {
+        description: 'Indicates if this auth input field is safe (not secret).',
+        type: 'boolean',
+      },
+    },
+
+    // Do not allow certain isSafe = True if sensitive tokens exists
+    not: {
+      properties: {
+        key: {
+          enum: [
+            'access_token',
+            'refresh_token',
+            'api_key',
+            'password',
+            'secret',
+          ],
+        },
+        isSafe: { const: true },
+      },
+      required: ['key', 'isSafe'],
+    },
+  },
+
```

Comment:
Let's add some examples and anti-examples specifically for authentication with the new `isSafe` property. We could add only for basic auth (email and password) and custom auth (api_key). OAuth auths don't always require these fields data from users.